src/screen/Council/MotionsScreen.tsx

```diff
@@ -0,0 +1,273 @@
+import {Button, Divider, Icon, Layout, Text, TopNavigationAction, useTheme} from '@ui-kitten/components';
+import React, {useContext, useMemo, useState} from 'react';
+import globalStyles, {standardPadding} from 'src/styles';
+import ScreenNavigation from 'layout/ScreenNavigation';
+import {NavigationProp} from '@react-navigation/native';
+import {ChainApiContext} from 'context/ChainApiContext';
+import {FlatList, StyleSheet, TouchableOpacity, View} from 'react-native';
+import type {Call} from '@polkadot/types/interfaces';
+import {FunctionMetadataLatest, ProposalIndex} from '@polkadot/types/interfaces';
+import {formatNumber} from '@polkadot/util';
+import {AccountContext} from 'context/AccountContextProvider';
+import {useVotingStatus} from '../../hook/useVotingStatus';
+import type {DeriveCollectiveProposal} from '@polkadot/api-derive/types';
+import {TxContext} from 'context/TxContext';
+import {EmptyView} from 'presentational/EmptyView';
+import Padder from 'presentational/Padder';
+import type {IExtrinsic, IMethod} from '@polkadot/types/types';
+import {Compact, GenericCall, getTypeDef} from '@polkadot/types';
+import {useCouncilMembers} from 'screen/Council/CouncilScreen';
+import {useQuery, useQueryClient} from 'react-query';
+import {Param, Params} from 'presentational/Params';
+
+export function MotionsScreen({navigation}: {navigation: NavigationProp<DashboardStackParamList>}) {
+  const {api} = useContext(ChainApiContext);
+  const {data, refetch, isLoading} = useQuery('motions', () => api?.derive.council.proposals());
```

Comment:
Are proposals and motions the same? Why are we calling it Motions?

Reply:
it's a bit confusing, as far as I could understand it. a proposal is called a motion in https://polkadot.js.org/ when it's ready to be voted on. for example you can go to the treasury and submit a proposal there. but it wouldn't show up on this call until you specifically move it to the council. 

I think some proposals can become a referenda and go through public voting, which is different from this.

I picked `motions` as the query key because it's what this page is called here https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkusama.api.onfinality.io%2Fpublic-ws#/council/motions which is showing this data

Reply:
it really is a bit confusing, I need to read more about this :D